src/embed-webfonts.ts

```
@@ -203,7 +203,7 @@ async function parseWebFontRules<T extends HTMLElement>(
 }
 
 function normalizeFontFamily(font: string) {
-  return font.trim().replace(/["']/g, '')
+  return font?.trim().replace(/["']/g, '')
```

return font?.trim()?.replace(/["']/g, '')

---

Well I'm not sure if it's needed, only font could be undefined. Trim and replace will not throw errors I guess if it's an empty string

---

only undefined will throw exception, and there are the possibilities of font coming as undefined.

---

I see, I cannot edit my code right now, I hope the maintainer could add this silent fail for fonts in a separate commit then